include/util/graph_loader.hpp

```diff
@@ -32,27 +33,17 @@ namespace util
  * The since the restrictions reference nodes using their external node id,
  * we need to renumber it to the new internal id.
 */
-inline unsigned loadRestrictionsFromFile(std::istream &input_stream,
+inline unsigned loadRestrictionsFromFile(std::string &filename,
```

Comment:
@MoKob the only reason I'm switching to filename is so that I can use the new fileread library that was merged in with @danpat's [big refactor PR](https://github.com/Project-OSRM/osrm-backend/pull/3165/files#diff-6404e45b65aab921c04657dd4505e125R27) which as far as I can tell only allows fileNames. 

I'll change the the filename into being a const.